pkgs/tools/security/quill-qr/default.nix

```diff
@@ -0,0 +1,53 @@
+{ bash
+, coreutils
+, fetchFromGitHub
+, gzip
+, jq
+, lib
+, makeWrapper
+, qrencode
+, stdenv
+, util-linux
+}:
+
+stdenv.mkDerivation rec {
+  pname = "quill-qr";
+  version = "e66726bda5177975bca11507214236d841eed32b";
+
+  src = fetchFromGitHub {
+    owner = "IvanMalison";
+    repo = "quill-qr";
+    rev = "${version}";
+    sha256 = "1kdsq6csmxfvs2wy31bc9r92l5pkmzlzkyqrangvrf4pbk3sk0r6";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -a quill-qr.sh $out/bin/quill-qr.sh
+  '';
+
+  wrapperPath = with lib; makeBinPath [
+    qrencode
+    coreutils
+    jq
+    gzip
+    util-linux
+  ];
+
+  fixupPhase = ''
+    patchShebangs $out/bin
+
+    wrapProgram $out/bin/quill-qr.sh --prefix PATH : "${wrapperPath}"
+  '';
```

Comment:
It might be nice to do this in the `installPhase` as well, as to overwrite the least phases as possible. Though that's completely subjective, feel free to ignore this comment.